src/Core/StorageManager.php

```diff
@@ -132,7 +141,8 @@ public static function move($dest, $tables = null)
 			$rr = DBA::select(
 				$table, 
 				['id', 'data', 'backend-class', 'backend-ref'],
-				['`backend-class` IS NULL or `backend-class` != ?' , $dest ]
+				['`backend-class` IS NULL or `backend-class` != ?' , $dest ],
+				['limit' => 5000]
```

Comment:
I guess it would be good to use the "id" field in the query and to store the latest processed table record id in a config variable.

Background: The field "backend-class" isn't part in any index, which means that there will be a lot of table scan needed, especially at the end, when most rows are already processed.

Reply:
True. But also it's something you'll need to run once or twice in all your node lifespan...